tools/mason/MasonExample.chpl

```diff
@@ -289,7 +289,9 @@ private proc runExampleBinary(projectHome: string, exampleName: string,
 
   const exampleResult = runWithStatus(command, true);
   if exampleResult != 0 {
-    throw new owned MasonError("Mason failed to find and run compiled example: " + exampleName + ".chpl");
+    throw new owned MasonError("Example has not been compiled: " + exampleName + ".chpl\
```

Comment:
How about this?

```chpl
    throw new owned MasonError("Example has not been compiled: " + exampleName + ".chpl\n" +
    "  Try running: mason build --example " + exampleName + ".chpl\n" +
    "           or: mason run --example " + exampleName + ".chpl --build");
```

Reply:
is it fine to remove the space between " and Try it doesn't look good that way

Reply:
The indent makes it clearer that the following line is a continuation of the previous error message